src/pages/blog/scale-of-usage.md

```diff
@@ -0,0 +1,71 @@
+
+## Artifact of infrastucture ##
+
+The closer you get to lower layers of intrastructure, the more important it becomes. Also, the greater impact one can have should details make it into the eyes of those who would do harm.
+
+Typically in communities, we can see the power lines, we might be able to figure out where the water lines are precisely, but the closer we get to those infrastucture centers, the more secretive or restrictive those environments need to be to ensure the service and security of the services being provided.
+
+It is the same for cloud infrastucture operators, telecommunications providers, and even train operators. They provide a substrate service to meet the needs of their customers. Often, the customer never knows how the server got there. Or how their telephone call completed, or what drove the spike into the railroad track tie.
+
+On some level, that is an implementation detail on a path.
+
+## It is hard for vendors as well ##
+
+It is also not the easiest thing for vendors who specialize in providing support or development services to even have these answers either. As an entity, they are bound by their agreements and consideration of the customer's privacy. Often reports and telemetry data, due to information security concerns, come from reproduction or internal test environments, and not actual production environments where an issue is being encountered which requires assistance or support. Ultimately, sometimes the hardest thing to wrap one's head around in these sorts of situations is that there is often not just a singular customer environment, but many.
+
+Vendors are in a very similar place to the contributors. They serve as proxies for their customers at the end of the day.
+
+## The end result ##
+
+The result of this is a lack of a clear picture of usage. It is difficult to truly understand the users of any given piece of Open Source software.
+
+But, we *can* piece together public context, and this is why the [OpenStack Baremetal Logo program](https://www.openstack.org/use-cases/bare-metal/) is helpful to the Ironic community. It helps highlight the firms which have an interest in the software. To join the program, each organization was required to submit their use case in how they use ironic. A number of them resulted in blog posts and stories. While this doesn't always result in explicit counts of number of servers, it is only the portion who actively stepped forward.
+
+Another avenue for guaging interest and usage is the actual upstream project is through who contributes. We don't necessarilly have context into thier usage or use case unless they speak of it publically, but a commit from a firm may be be driven by any number of different drivers: Customer needs, Internal Need, Bugfixes from internal or external usage, Compelled need to fix a typo, or a friend just wanting to make sure they can have breakfast on occassion with another friend resulting in a pain point being fixed.
```

Comment:
Nit: is who contributes to the actual upstream project.

Reply:
Good point, done.